main/realmain.cc

```diff
@@ -950,19 +950,10 @@ int realmain(int argc, char *argv[]) {
             FileOps::write(opts.storeState.c_str(), core::serialize::Serializer::store(*gs));
         }
 
-        auto untypedSources = getAndClearHistogram("untyped.sources");
-        if (opts.suggestSig) {
-            ENFORCE(sorbet::debug_mode);
-            vector<pair<string, int>> withNames;
-            long sum = 0;
-            for (auto e : untypedSources) {
-                withNames.emplace_back(core::SymbolRef::fromRaw(e.first).showFullName(*gs), e.second);
-                sum += e.second;
-            }
-            fast_sort(withNames, [](const auto &lhs, const auto &rhs) -> bool { return lhs.second > rhs.second; });
-            for (auto &p : withNames) {
-                logger->error("Typing `{}` would impact {}% callsites({} out of {}).", p.first, p.second * 100.0 / sum,
-                              p.second, sum);
+        if constexpr (sorbet::track_untyped_blame_mode) {
+            if (opts.printBlameUntyped) {
+                auto untypedBlames = getAndClearHistogram("untyped.blames");
```

Comment:
I think we should move this outside the `if (opts.printBlameUntyped)`.

If someone built sorbet in this custom release mode, but forgot to pass this print option, we're not going to call `getAndClearHistogram`, which is going to spam statsd with a high-cardinality metric, which is not good.